packages/contracts/contracts/protocol/incentives/ExternalRewardDistributor.sol

```diff
@@ -34,7 +35,9 @@ contract ExternalRewardDistributor is IExternalRewardsDistributor, Initializable
 
   address public rewardAdmin;
 
-  uint256[40] __gap_ExternalRewardDistributor;
+  address public curveGaugeFactory;
```

Comment:
I think Curve calls this contract the "minter" contract: https://curve.readthedocs.io/dao-gauges.html#minter. Not necessary to change

Reply:
No, they call it https://github.com/curvefi/curve-xchain-factory/blob/master/contracts/ChildGaugeFactory.vy